tensorbay/client/dataset.py

```diff
@@ -220,10 +220,11 @@ def cache_enabled(self) -> bool:
     def squash_and_merges(self) -> SquashAndMerge:
         """Get class :class:`~tensorbay.client.version.SquashAndMerge`.
 
-        Return:
+        Returns:
             Required :class:`~tensorbay.client.version.SquashAndMerge`.
 
         """
+        return SquashAndMerge(self._dataset_id, self._client, self._status)
```

Comment:
`squash_and_merges` or `squash_and_merge`?

Reply:
shall we cache the `SquashAndMerge` instance?

Reply:
Here will be modified after all implementations are complete. The `squash_and_merge` function is not deleted yet.

Reply:
what means cache the instance?

Reply:
here every time user calls `squash_and_merges` will create a `SquashAndMerge` instance, shall we cache it so that the second time user calls it we do not need to create it again.